draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -540,6 +540,10 @@ defined:
 * `extends`
   * OPTIONAL. A URI of another type that this type extends, as described in
   (#extending-type-metadata).
+* `display`: An object containing display information for the type, as described
+  in (#display-metadata). This property is OPTIONAL.
+* `claims`: An object containing claim information for the type, as described in
+  (#claim-metadata). This property is OPTIONAL.
```

Comment:
at some point (not necessarily this PR tho) an example type metadata document showing some/many of these options would be very helpful

Reply:
i know there's something over in https://github.com/vcstuff/sd-jwt-vc-types

Reply:
duly noted by me